### ISSUE TITLE
Deletes Posts

### ISSUE THREAD

**Issue by ghost** [Darrenmeehan/BuddyPress-Like#47]
If I like a post with this enabled it deletes the post from the database

**Comment by aswain1** [comment]
I also experienced this bug, but only if I "unlike" a post. The unlike deletes the activity, comment, etc. completely. This bug forced me to look elsewhere for the "Like" functionality.

**Comment by aswain1** [comment]
Interestingly, getting the .zip file from Github and installing it that way fixes the bug. It's only when I install from the Wordpress Plugin Directory that I run into that issue.

**Comment by ghost** [comment]
Hey thanks!

**Comment by manu89ft** [comment]
Is there a bug on the wordpress repository code? Are we sure?

**Comment by 13pixlar** [comment]
@manu89ft Yes. 100% sure. Its a fatal bug that deletes all the users posts after like/dislike own post.

**Comment by phylaxis** [comment]
Has this been patched in the WP repository code yet?
